src/placeos-frontend-loader/api/remotes.cr

```diff
@@ -0,0 +1,45 @@
+module PlaceOS::FrontendLoader::Api
+  class Remotes < Base
+    base "/api/frontend-loader/v1/remotes"
+    Log = ::Log.for(self)
+
+    private alias Remote = PlaceOS::FrontendLoader::Remote
+
+    # Returns an array of releases for a repository
+    get "/:repository_url/releases", :releases do
```

Comment:
+ a tags endpoint

and gotta update the client too  🧠 🔨

would be nice if we had autogenerated RPCs 🤔